app/views/pages/articles/2016-11-17-ruby-au-elections.html.md

```diff
@@ -0,0 +1,48 @@
+# Ruby Au Elections
+
+
+We have 5 positions open, President and Secretary, as well as two full-term and one half-term general position. Below is a brief description of each position, and a link to a nomination form to help streamline the meeting. If you are nominating someone other than yourself, please be sure to check with them first.
+
+Each committee member is expected to attend a monthly 1 hour online meeting (scheduled around all members other commitments), and to have regular access to Slack, 1password and Google docs. These are minimal requirements for staying in contact with the rest of the committee and accessing required files and login information. The time outputs outlined below are loose, with more time often needed at crucial points throughout the year and less at other times. Critical points are usually: turnover of committee members (after each Rails Camp), the weeks before RubyConf and Rails Camps and the turnover of the financial year.
+
+
+[nominations](https://docs.google.com/forms/d/e/1FAIpQLSdK7a3N3-5XT5LRqZIVBqEW1NgcxNTZJqaVs7kHwgwyZ1wDog/viewform)
+
+
+## President
+The president guides the overall direction of the organisation. They make the final call on major decisions and delegate work to general members. They are in charge of making sure the committee is running smoothly and committee members are meeting their obligations. Occasionally they need to mediate interpersonal issues within the committee or address code of conduct breaches within the member base.
+
+The president is in charge of chairing the general meetings, though this is often delegated to a trusted member of the organisation. They also cast the deciding vote in the instance of a draw.
```

Comment:
> They also cast the deciding vote in the instance of a draw

How about: when the Committee votes and there's a draw, the President has the deciding vote.